packages/genesys-spark-components/src/components/beta/gux-icon-tooltip/gux-icon-tooltip.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxIconTooltipVariant } from './gux-icon-tooltip.types';
+
+/**
+ * @slot content
+ */
+
+@Component({
+  styleUrl: 'gux-icon-tooltip.scss',
+  tag: 'gux-icon-tooltip-beta',
+  shadow: true
+})
+export class GuxIconTooltip {
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxIconTooltipVariant = 'info';
+
+  private getVariantIcon(variant: GuxIconTooltipVariant): string {
+    switch (variant) {
+      case 'question':
+        return 'fa/circle-question-regular';
+      default:
+        return 'fa/circle-info-regular';
+    }
+  }
+
+  componentWillLoad(): void {
+    trackComponent(this.root, { variant: this.variant });
+  }
+
+  render(): JSX.Element {
+    return (
+      <button type="button" class="gux-icon-tooltip">
+        <gux-icon
+          icon-name={this.getVariantIcon(this.variant)}
+          size="small"
+          decorative
```

Comment:
Made this decorative since the button itself is described by the tooltip so didn't want to duplicate.